numba/core/typing/setdecl.py

```diff
@@ -97,18 +67,6 @@ def _resolve_xxx_update(self, set, args, kws):
         if isinstance(iterable, types.Set) and iterable.dtype == set.dtype:
             return signature(types.none, iterable)
 
-    @bound_function("set.difference_update")
-    def resolve_difference_update(self, set, args, kws):
-        return self._resolve_xxx_update(set, args, kws)
```

Comment:
Is `self._resolve_xxx_update` now dead code?

Reply:
Yes, it is! I'll remove it